PLD runs etckeeper and other hooks from /etc/poldek/{pre,post}-install.d.

diff -Nur a/conf/Makefile.am b/conf/Makefile.am
--- a/conf/Makefile.am	2026-10-25 20:33:16.000000000 +0100
+++ b/conf/Makefile.am	2026-10-25 21:14:51.730295076 +0100
@@ -26,6 +26,10 @@
 
 install-data-local: $(GENCONFIGS)
 	 $(MKDIR_P) $(DESTDIR)$(sysconfdir)/$(PACKAGE)/repos.d
+	 $(MKDIR_P) $(DESTDIR)$(sysconfdir)/$(PACKAGE)/pre-install.d
+	 $(INSTALL_DATA) pre-install.d/README $(DESTDIR)$(sysconfdir)/$(PACKAGE)/pre-install.d
+	 $(MKDIR_P) $(DESTDIR)$(sysconfdir)/$(PACKAGE)/post-install.d
+	 $(INSTALL_DATA) post-install.d/README $(DESTDIR)$(sysconfdir)/$(PACKAGE)/post-install.d
 	 for a in $(CONFIGS); do \
 	 	$(INSTALL_DATA) $(srcdir)/$$a $(DESTDIR)$(sysconfdir)/$(PACKAGE); \
 	 done
diff -Nur a/conf/poldek.conf b/conf/poldek.conf
--- a/conf/poldek.conf	2026-10-25 21:14:42.203896190 +0100
+++ b/conf/poldek.conf	2026-10-25 21:14:42.249524757 +0100
@@ -70,7 +70,7 @@
 # run in the order of their names. Hooks do not run for rpm test runs nor
 # with a root directory other than /. Example:
 # hooks dir = /etc/poldek
-#hooks dir = 
+hooks dir = /etc/poldek
 
 # Transaction summary style, can be 'old' (pre 0.4.0 version) or 'color' - coloured and compact.
 #summary style = color
diff -Nur a/conf/post-install.d/README b/conf/post-install.d/README
--- a/conf/post-install.d/README	1970-01-01 01:00:00.000000000 +0100
+++ b/conf/post-install.d/README	2026-10-25 21:14:51.659260033 +0100
@@ -0,0 +1,3 @@
+Executable files in this directory are run, in name order, after poldek
+has installed, upgraded or removed packages.
+See "hooks dir" in /etc/poldek/poldek.conf.
diff -Nur a/conf/pre-install.d/README b/conf/pre-install.d/README
--- a/conf/pre-install.d/README	1970-01-01 01:00:00.000000000 +0100
+++ b/conf/pre-install.d/README	2026-10-25 21:14:51.659260033 +0100
@@ -0,0 +1,3 @@
+Executable files in this directory are run, in name order, before poldek
+installs, upgrades or removes packages; if one of them fails, poldek stops.
+See "hooks dir" in /etc/poldek/poldek.conf.
